languages/wikipron/src/codes.py

```diff
@@ -21,6 +21,7 @@
 * Dialect information may also need to be added manually
 """
 
+# TODO Generate and use a lookup table for the iso639-3 tsv file
```

Comment:
Generic minor comment throughout this change: `TSV` when you're talking about it in a comment (okay to have a variable name like `tsv` cuz snake_casing conventions).